### PR TITLE
Removed facades lang and url from views

### PR DESCRIPTION
**Pull request by joecohens**
https://github.com/cachethq/Cachet/issues/217

### CODE REVIEW

File: app/views/dashboard/components/add.blade.php [hunk 1]
```diff
@@ -51,8 +51,8 @@
                     </fieldset>
 
                     <button type="submit" class="btn btn-success">Submit</button>
-                    <a class="btn btn-default" href="{{ URL::route('dashboard.components') }}">Cancel</a>
-                    <input type='hidden' name='component[user_id]' value='{{ Auth::user()->id }}' />
+                    <a class="btn btn-default" href="{{ route('dashboard.components') }}">Cancel</a>
+               <input type='hidden' name='component[user_id]' value='{{ Auth::user()->id }}' />
```
Comment (GrahamCampbell) [inline app/views/dashboard/components/add.blade.php]: Why's the indentation level changed here?

**Comment by GrahamCampbell** [comment]
Asside from my minor comment, :+1:

File: app/views/dashboard/components/add.blade.php (hunk 1 above)
Comment (joecohens) [inline app/views/dashboard/components/add.blade.php]: No idea haha maybe the find and repl.. I'll fix that.

**Comment by GrahamCampbell** [comment]
Awesome. :)

**Comment by joecohens** [comment]
Fixed going to merge this :)

**Comment by joecohens** [comment]
Lol my internet got slow thanks :)

**Pull request merged**